app/helpers/shared_helper.rb

```diff
@@ -1,16 +1,6 @@
 # frozen_string_literal: true
 
 module SharedHelper
-  def distributor_link_class(distributor)
```

Comment:
The reason for removing this method can be better understood through this commit message:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12324/commits/ac1595e718fd1bf15eeec1503fa6bdcc4fa83489

Reply:
Good find!